Apps/Src/ReadSwitches.c

```diff
@@ -0,0 +1,128 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+// Includes
+#include "ReadSwitches.h"
+
+// Macros
+#define OS_DELAY_MS(ms) \
+    OSTimeDlyHMSM(0, 0, 0, (ms), OS_OPT_TIME_HMSM_NON_STRICT, &err)
+#define MAX_SWITCH_LAG          100  // delay (ms) for checking switches while precharging
```

Comment:
This is just the interval we want to check the ignition switch for changes during array precharge right?